lib/index.js

```diff
@@ -48,6 +48,7 @@ export default class Layer extends EE.EventEmitter {
     this.click = opts.click;
     this.type = type;
     this.rules = opts.rules;
+    this.cacheBust = opts.cacheBust || false;
```

Comment:
```suggestion
    this.cacheBust = opts.cacheBust;
```

not actually necessary since falsey values are okay here